docs/src/main/asciidoc/spring-cloud-config.adoc

```diff
@@ -893,6 +905,99 @@ Configuration files are stored in your bucket as `{application}-{profile}.proper
 
 NOTE: When no profile is specified `default` will be used.
 
+==== AWS Parameter Store Backend
+
+Spring Cloud Config Server supports AWS Parameter Store as a backend for configuration properties. You can enable this feature by adding a dependency to the link:https://github.com/aws/aws-sdk-java/tree/master/aws-java-sdk-ssm[AWS Java SDK for SSM].
+
+[source,xml,indent=0]
+.pom.xml
+----
+    <dependency>
+        <groupId>com.amazonaws</groupId>
+        <artifactId>aws-java-sdk-ssm</artifactId>
+    </dependency>
+----
+
+The following configuration uses the AWS SSM client to access parameters.
+
+[source,yaml]
+----
+spring:
+  profiles:
+    active: awsparamstore
+  cloud:
+    config:
+      server:
+        awsparamstore:
+          region: eu-west-2
+          endpoint: https://ssm.eu-west-2.amazonaws.com
+          origin: aws:parameter:
+          prefix: /config/service
+          profileSeparator: _
+          recursive: true
+          decryptValues: true
+          maxResults: 5
+----
+
+The following table describes the AWS Parameter Store configuration properties.
+
+.AWS Parameter Store Configuration Properties
+|===
+|Property Name |Required |Default Value |Remarks
+
+|*region*
+|no
+|
+|The region to be used by the AWS Parameter Store client. If it's not explicitly set, the SDK tries to determine the region to use by using the link:https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/java-dg-region-selection.html#default-region-provider-chain[Default Region Provider Chain].
+
+|*endpoint*
+|no
+|
+|The URL of the entry point for the AWS SSM client. This can be used to specify an alternate endpoint for the API requests.
+
+|*origin*
+|no
+|`aws:ssm:parameter:`
+|The prefix that is added to the property source's name to show their provenance.
+
+|*prefix*
+|no
+|`/config`
+|Prefix indicating L1 level in the parameter hierarchy for every property loaded from the AWS Parameter Store.
+
+|*profileSeparator*
```

Comment:
`profile-separator`. We should refer to properties in kebab-case in docs.

Reply:
Happy to oblige. But I think the suggested changes are not consistent with the existing docs format as camel case is used in these places and a few others:
https://github.com/spring-cloud/spring-cloud-config/blob/615f7521b4235bb8f1e11d8f333d95df1c3ef7b7/docs/src/main/asciidoc/spring-cloud-config.adoc#L416-L434
https://github.com/spring-cloud/spring-cloud-config/blob/615f7521b4235bb8f1e11d8f333d95df1c3ef7b7/docs/src/main/asciidoc/spring-cloud-config.adoc#L605-L614
https://github.com/spring-cloud/spring-cloud-config/blob/615f7521b4235bb8f1e11d8f333d95df1c3ef7b7/docs/src/main/asciidoc/spring-cloud-config.adoc#L722

Reply:
@spencergibb @maciejwalkowiak ☝️